charts/rancher-backup/rancher-backup/2.0.0/Chart.yaml

```diff
@@ -9,7 +9,6 @@ annotations:
   catalog.cattle.io/scope: management
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: rancher-backup
-  catalog.cattle.io/rancher-version: < 2.6.1-0
```

Comment:
this should NOT be dropped. 
I doubt your base is not the latest release-v2.6 ?

Reply:
It is. It's the same issue `max version` again.
https://github.com/rancher/charts/pull/1541#pullrequestreview-775444057